sample-extension/EdFi.Ods.Extensions.Sk/EdFi.Ods.Extensions.Sk.nuspec

```diff
@@ -16,6 +17,6 @@
     <file src="Artifacts/**/*.sql" target="."/>
     <file src="Artifacts/**/*.json" target="."/>
     <file src="assemblyMetadata.json" target="."/>
-    <file src="bin/$configuration$/net6.0/EdFi.Ods.Extensions.Sk.dll" target="."/>
+    <file src="bin/$Configuration$/net6.0/EdFi.Ods.Extensions.Sk.dll" target="."/>
```

Comment:
Is this change needed? I would have thought the previous way it was specified with a lowercase c would also work.

Reply:
yes I worked for lowercase